docs/news/unreleased/edge-container-nginx.md

```diff
@@ -0,0 +1,8 @@
+# RHEL-Edge container image now uses nginx
```

Comment:
As this is a breaking change, I think we should mention it in `news`. Otherwise, this PR looks good. :)

Reply:
Good point.  I updated the news item to mention mostly the things that a user would care about (port change).